packages/adapter-mongoose/lib/adapter-mongoose.js

```diff
@@ -317,13 +328,21 @@ class MongooseFieldAdapter extends BaseFieldAdapter {
   }
 
   mergeSchemaOptions(schemaOptions, { mongooseOptions }) {
+    // JM 190719:
+    // It seem to me this implementation will cause us to accumulate indexes
+    // Will indexes that are removed from code be dropped?
```

Comment:
Yes, they will - Mongoose has an index-syncing code that we leverage which will drop existing indexes and add new ones.

```suggestion
```